Cargo.toml

```diff
@@ -31,17 +31,18 @@ clap = { version = "4.5.48", features = [
 ] }
 config = "0.15.17"
 crossterm = { version = "0.28.1", features = ["serde", "event-stream"] }
-delegate = "0.13"
 derive_deref = "1.1.1"
 directories = "6.0.0"
 either = "1.15"
+embedded-graphics-simulator = "0.7.0"
```

Comment:
Could we feature gate this behind a `simulator` feature flag?
Ideally you would just `cargo run --features simulator`. And we wouldn't need CLI flags at all.